packages/apputils-extension/src/index.ts

```diff
@@ -611,11 +611,38 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
  * The default HTML sanitizer.
  */
 const sanitizer: JupyterFrontEndPlugin<ISanitizer> = {
-  id: '@jupyter/apputils-extension:sanitizer',
+  id: '@jupyterlab/apputils-extension:sanitizer',
   autoStart: true,
   provides: ISanitizer,
-  activate: () => {
-    return defaultSanitizer;
+  requires: [ISettingRegistry],
+  activate: (app: JupyterFrontEnd, settings: ISettingRegistry) => {
+    const userSanitizer: Sanitizer = new Sanitizer();
+    const loadSetting = (setting: ISettingRegistry.ISettings): void => {
+      const schemesArray = setting.get('additionalSchemes')
+        .composite as Array<string>;
+
+      if (userSanitizer.setSchemes) {
+        userSanitizer.setSchemes(schemesArray);
+      }
```

Comment:
This is almost complete now! The visual regression tests are failing because valid links are no longer shown and instead get sanitised away. This is because the schemes which were previously implicitly allowed by default (`[ 'http', 'https', 'ftp', 'mailto', 'tel' ]`) are no longer allowed. I would suggest to rename  `additionalSchemes` to `allowedSchemes` and in the JSON schema add a default of `[ 'http', 'https', 'ftp', 'mailto', 'tel' ]`; I would also suggest setting this array on `allowedSchemes` instead of an empty one.

Reply:
In case if you wonder, this comes from upstream from [here](https://github.com/apostrophecms/sanitize-html/blob/ba3a2f67e3fdcefb7212b8776111ad6f248b52cb/index.js#L814)

Reply:
The alternative is to keep `additionalSchemes` as is but make `setSchemes` concatenate the default (from an immutable source of truth) with the additional. In that case it would probably make sense to name it `setAdditionalSchemes`. The former approach however also allows users to remove existing schemes giving them more control (e.g. I can see why someone could remove `http` but keep `https`).